### PR TITLE
e2e-GroupBy Test Scenarios

### PR DESCRIPTION
**Pull request by itsmekumari**
@sawantpritam Please review.

### CODE REVIEW

File: e2e-test/src/test/java/io/cdap/plugin/groupby/stepsdesign/GroupByAnalytics.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  @Then("Validate OUT record count of groupby is equal to IN record count of sink")
+  public void validateOUTRecordCountOfGroupByIsEqualToINRecordCountOfSink() {
+    Assert.assertEquals(recordOut(), GroupByActions.getTargetRecordsCount());
+  }
+
+  @Then("Enter GroupBy plugin Fields {string}")
+  public void enterGroupByPluginFields(String jsonFields) {
+    GroupByActions.enterFields(jsonFields);
+  }
+
```
Comment (sawantpritam) [inline e2e-test/src/test/java/io/cdap/plugin/groupby/stepsdesign/GroupByAnalytics.java]: Remove empty lines.

Check all files.

**Review by sawantpritam — CHANGES_REQUESTED** [review]

File: e2e-test/src/test/features/GroupByMacros.feature [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@GroupBy
+Feature:GroupBy - Verification of GroupBy pipeline with GCS as source and File as target using macros
+
+  @GCS_CSV_GROUPBY_TEST @FILE_SINK_TEST
+  Scenario: To verify data is getting transferred from GCS to File successfully with GroupBy plugin properties as macro arguments
```
Comment (sawantpritam) [inline e2e-test/src/test/features/GroupByMacros.feature]: Can we use File Source and sink in all scenarios ? so that there wont be rework incase.

**Review by sawantpritam — CHANGES_REQUESTED** [review]

File: e2e-test/src/test/java/io/cdap/plugin/groupby/stepsdesign/GroupByAnalytics.java (hunk 1 above)
Comment (itsmekumari) [reply e2e-test/src/test/java/io/cdap/plugin/groupby/stepsdesign/GroupByAnalytics.java]: Removed, checked all files.

File: e2e-test/src/test/features/GroupByMacros.feature (hunk 2 above)
Comment (itsmekumari) [reply e2e-test/src/test/features/GroupByMacros.feature]: Used only File as source and File as sink in scenarios.

**Pull request closed without merging**